### PR TITLE
Bug 1833486: Improve fluent performance in comparison to 4.4 

### PR DESCRIPTION
**Pull request by jcantrill**
This PR addresses the performance variance from 4.4:

* Reverts the change to the number of queued chunks: #479
* Cap disk usage based an a percent and total_byte_limit
* modify the rotate_wait parameter to 1.0
* Configure pos_file_compaction_interval to 5m to address

https://bugzilla.redhat.com/show_bug.cgi?id=1833486

### CODE REVIEW

**Comment by jcantrill** [comment]
@vimalk78 can you do a quick review to ensure the syslog changes were not accidently reverted during my merge attempts?

File: files/fluentd/run.sh [hunk 1]
```diff
@@ -86,34 +88,29 @@ if [ $DF_LIMIT -eq 0 ]; then
...
+TOTAL_LIMIT_SIZE=$(expr $ALLOWED_DF_LIMIT / $NUM_OUTPUTS) || :
+echo "Setting each total_size_limit for $NUM_OUTPUTS buffers to $TOTAL_LIMIT_SIZE bytes"
+export TOTAL_LIMIT_SIZE
+
+##
+# Calculate the max number of queued chunks given the size of each chunk
+# and the max allowed space per buffer
+## 
+BUFFER_SIZE_LIMIT=$(echo ${BUFFER_SIZE_LIMIT:-8m} |  sed -e "s/[Kk]/*1024/g;s/[Mm]/*1024*1024/g;s/[Gg]/*1024*1024*1024/g;s/i//g" | bc)
+BUFFER_QUEUE_LIMIT=$(expr $TOTAL_LIMIT_SIZE / $BUFFER_SIZE_LIMIT)
```
Comment (vimalk78) [inline files/fluentd/run.sh:111]: shouldnt we divide the `TOTAL_LIMIT_SIZE` into two parts? one for chunks and another for queue, and the calculate the queue size based on `TOTAL_LIMIT_SIZE / 2` ?

**Pull request merged**